src/services/doppler-legacy-client.ts

```diff
@@ -259,6 +299,74 @@ function mapNavMainEntry(json: any): MainNavEntry {
   };
 }
 
+function sanitizePlans(json: any): any {
+  return json.length ? json.filter((rawPlan: any) => rawPlan.Fee) : [];
```

Comment:
I'm not sure what this function do, but, what if plan doesn't have fee?

Reply:
Exclusive plans as we use today have no fee. We want to erradicate them.

Reply:
And what about prepaids?
`{ type: 'prepaid', id: 1, name: '1500-CREDITS', credits: 1500, price: 5303 },`

Reply:
You are viewing the data in the double service, and that has been sanitized already. Fee is transformed into price here https://github.com/FromDoppler/doppler-webapp/pull/1054/files#diff-454cb41308aec9b948f8c8be1c591e4cR365

Reply:
If the input is already sanitized, maybe you should accept a typed data in place of _json: any_, is it possible?

Reply:
The data is sanitized in `sanitizePlans` function. It gets here raw, that is why is a type any. @lucianaDilizio asked about the data in the double service, that is hardcoded sanitized here https://github.com/FromDoppler/doppler-webapp/pull/1054/files#diff-58ff07f0057a676a67be774807741de9R47